tools/buildtools/ambertools/parmconv.xml

```diff
@@ -177,10 +192,8 @@ for system in systems:
       <sanitizer>
         <valid initial="string.printable">
           <remove value="&amp;"/>
+          <add value="&amp;"/>
         </valid>
-        <mapping initial="none">
-          <add source="&amp;" target="__and__"/>
-        </mapping>
```

Comment:
I didn't really get why this mapping was added? I think we want to add the ampersand, not map it.

Reply:
:+1:

Reply:
@simonbray  Do complex selections work for you after changing this? I had an with the following 1-500@O&!(:WAT|:LYS,ARG)
Maybe include the unescape function from Bjoern in parmconv.py?
```
def unescape(cond_text):
    """
    # Unescape if input has been escaped - credit @bgruening //github.com/bgruening/galaxytools.git get_online_data.py
    """
    mapped_chars = { '>' :'__gt__',
                 '<' :'__lt__',
                 "'" :'__sq__',
                 '"' :'__dq__',
                 '[' :'__ob__',
                 ']' :'__cb__',
                 '{' :'__oc__',
                 '}' :'__cc__',
                 '@' : '__at__',
                 '\n' : '__cn__',
                 '\r' : '__cr__',
                 '\t' : '__tc__',
                 '&' : '__and__'
                 }
    for key, value in mapped_chars.items():
        cond_text = cond_text.replace( value, key )
    return cond_text
```

and then 
```
if args.istripmask is not None:                                     
    if args.istripmask == "":
        pass
    else:
        gmx_top.strip(unescape(args.istripmask))
        


```